src/test/java/reposense/authorship/FileInfoExtractorTest.java

```diff
@@ -163,6 +168,29 @@ public void generateFileInfo_fileWithoutSpecialCharacters_correctFileInfoGenerat
         Assert.assertEquals(5, fileInfo.getLines().size());
     }
 
+    @Test
+    public void getListOfNonBinaryFiles_directoryWithBinaryAndNonBinaryFiles_capturesNonBinaryFiles() {
```

Comment:
`getListOfNonBinaryFiles` -> `getNonBinaryFiles`
`directoryWithBinaryAndNonBinaryFiles` -> `directoryWithBinaryFiles` - the presence of non binary files is implied.
`capturesNonBinaryFiles` -> `success` - no need explain result if its just a success or failure procedure

Reply:
> capturesNonBinaryFiles -> success - no need explain result if its just a success or failure procedure

That was my suggestion. Sorry didn't see clearly. 👀